packages/extension-sdk/src/util/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
```

Comment:
Are we automatically generating the wrong year? Or does this placate the linter?